main/acle.md

```diff
@@ -11821,6 +12032,143 @@ element types.
   svint8x4_t svuzpq[_s8_x4](svint8x4_t zn) __arm_streaming;
   ```
 
+### SME2.1 instruction intrinsics
+
+The specification for SME2.1 is in
+[**Alpha** state](#current-status-and-anticipated-changes) and may change or be
```

Comment:
change "may" to "might"

Reason? "may" has several meanings so is ambiguous.
See https://confluence.arm.com/display/TCOM/Concise+and+clear+words+and+phrases